### PR TITLE
Replace JS logo to be the official logo, adjust sass logo to be a similar size

### PR DESCRIPTION
**Pull request by ldamon**
Based off of this comment:
https://twitter.com/basecode/status/635790016980066305

screen shot 2015-09-01 at 11 11 01 am

### CODE REVIEW

File: app/assets/images/languages/javascript.svg [hunk 1]
```diff
@@ -1,27 +1,12 @@
...
...+w9+J68vGwvPklUQx6mvvLJkLRWgUHRv8Xzna9GtXQp1G7WIilXggEvf43w7Ag4Pzqx+r36Ls5ApY3+aQsWa9meejT3PLD72KgdMvmzsop8586/AldRYHAarUwDS+vn8CqT88TtO30iZVY6gVDV/d7rr3w1SwFXK/b1ut32Rj6i67L6Ds0tUvJvWLQLD8SjGF6Fxa0A6c2e073wiGAy8jg8uAnsGExkTnQRIgFIcDhkeNqpwlB6VIIVSmHgyV7qdHrG4xvn8BgcdmzLjqLMCC9adnnQmRHIpguYMUA97CZdZ8sOW/HNexpplZixFIvvE4wgTd3CYQbUev3B+Ub9/yo2tDPwK/FTYNicxR21odImjx/jsBzfBQysywDDw/6/MQvkiaUOqaJE0u9YCQYz1M/blmrnyvo1fH6/ax+w56U/Zqe+yX1UHqrYSQYxxWP8dwD5+U1e6g3adnLXTHVFdi0+jTE+3kvyR5KI5J9I5b1ikBX7S7L9rfKwuK2LA2TDEuucsLw7xXZf5nmUNbXeDT/u8HkvZbtnKVKB06XxECMRmr6pAQBx1Ws4TdVVf1dpx7e4raSCWJZbySMqe+DxrJNFpZPgx9k6uZi8FxPwI+8qijCa92eDF2PB/3fGQLgnuFhxzbOUqTV4+DlNBAkA9sTZmWl8DtJ9PraO2lWQ8wRy3olXW5yt+sGx7lO+XQYLk2sf8bK/+izwrCxgvOvSqK/vxNHw6zonlU9nGDiQcfxT+L8lVNyml3Mip2ux3umU3xPlvXXJNF9v59DnY0wZZpYGpWHXb13Ok5wkuf7IyK3cqKiuFqSE3lIIrYrVwKu+GdREktEJI01xN41zsm4njDlus6xfFA+Iac5efa0bK4RrOi4lqMtiFLhT5LEfSjzpdRM+KeKWJrB73Gj/+EH8nAQ8Irr2sdyvnFMEAQi+KkRZRGIRwikjbrG6NTG8wTX9UTH80WD59GznlQWRG2/JEozHA97Qvn5G5qVT/fShwD2aDzXn/I8G9qKMwJHCUb7bQUuC1uOPsfz0pIsa28Jgrco8fO7+q1Xt+Wnnli6NZzyEQKNCDjB+B7PDY7yOUEOfC8f+MZRimQN8XztiIGw0QuqUV79Gocwvs/D+4v3XFdd5gR9RhCkFZ73HFHkZ2V+bkc9bVa+/x+t8IuE23nIRQAAAABJRU5ErkJggg=="></image>
```
Comment (smharley) [inline app/assets/images/languages/javascript.svg]: this doesn't look like an svg anymore. looks like an image embedded in an svg, is there a vector version of this we could use?

Comment (ldamon) [inline app/assets/images/languages/javascript.svg]: aaaah so crazy

Comment (ldamon) [inline app/assets/images/languages/javascript.svg]: I was adjusting the one they have to fit with the style of the other logos...and used bitmap for the letters :shame:
I'll make this less gross.

**Comment by gylaz** [comment]
@ldamon We try to keep the commit title to under 50 characters. How about `Replace JS log and resize Sass logo`? This looks great!

**Comment by smharley** [comment]
LGTM if you make the change @gylaz suggested!

**Comment by ldamon** [comment]
screen shot 2015-09-01 at 11 11 01 am
new screenshot - looks the same to me?

**Comment by smharley** [comment]
:+1:

**Pull request merged**